libcontainer/cgroups/fscommon/rdma_test.go

```diff
@@ -0,0 +1,83 @@
+package fscommon
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+/* Roadmap for future */
+// (Low-priority)  TODO: Check if it is possible to virtually mimic an actual RDMA device.
+// TODO: Think of more edge-cases to add.
+
+// TestRdmaSet performs an E2E test of RdmaSet(), parseRdmaKV() using dummy device and a dummy cgroup file-system.
+// Note: Following test does not gurantees that your host supports RDMA since this mocks underlying infrastructure.
+func TestRdmaSet(t *testing.T) {
+	tempDir, err := ioutil.TempDir("", "cgroup_test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	root := tempDir
+	testCgroupPath := filepath.Join(root, "rdma")
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
what error are you trying to check here?

Reply:
@kolyshkin this is not needed. I don't remember why did i write it. :sob: . Resolved in latest commit.